test/utils/TestWorld.ts

```diff
@@ -36,6 +46,14 @@ export function defaultWorldFixture(): Promise<World> {
   });
 }
 
+export function baseWorldFixture(): Promise<World> {
```

Comment:
base vs default world is sort of confusing. Would be worth clarifying what you mean here.